quantecon/markov/approximation.py

```diff
@@ -12,11 +12,11 @@
 from numba import njit
 
 
-def rouwenhorst(n, ybar, sigma, rho):
+def rouwenhorst(n, rho, sigma, mu=0.):
```

Comment:
@Smit-create while it is difficult in this change to preserve backward compatibility, can you add a `deprecation` notice to the doctoring at the bottom.

Reply:
Done, thanks!